Assets/PatchKit Patcher/Scripts/PatchKitLogo.cs

```diff
@@ -0,0 +1,82 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using System.Collections;
+using UniRx;
+using PatchKit.Unity.Patcher.Debug;
+
+namespace PatchKit.Unity
+{
+
+    public class PatchKitLogo : MonoBehaviour
+    {
+        public string patchKitWebsiteUrl;
+
+        public Texture2D cursorTexture;
+
+        public Vector2 cursorHotspot;
+
+        public Image image;
+
+        public Button button;
+
+        private bool _isLogoVisible = false;
+
+        private const float LogoVisibilityChangeDelay = 3; // in seconds
+
+        private void Awake()
+        {
+            Assert.IsNotNull(button);
+            Assert.IsNotNull(image);
+
+            Assert.IsNotNull(cursorTexture);
+
+            button.enabled = false;
+            image.enabled = false;
+        }
+
+        private void Start()
+        {
+            var patcher = Patcher.Patcher.Instance;
+
+            Assert.IsNotNull(patcher);
+
+            patcher.AppInfo
+                .ObserveOnMainThread()
+                .Select(app => app.PatcherWhitelabel)
+                .Subscribe(Resolve)
+                .AddTo(this);
+
+            StartCoroutine(ChangeVisibility());
+        }
+
+        private IEnumerator ChangeVisibility()
+        {
+            while (true)
```

Comment:
Check this out -
 http://www.introtorx.com/Content/v1.0.10621.0/13_TimeShiftedSequences.html#Delay

Maybe it's possible to use it instead of coroutine.